src/modules/packages.c

```diff
@@ -53,6 +53,34 @@ static uint32_t getNumStrings(const char* filename, const char* needle)
     return count;
 }
 
+static void getActiveBranch()
+{
+    FILE* file = fopen("/etc/pacman-mirrors.conf", "r");
+    if(file == NULL)
+        return; // No file or not Manjaro[based]
+
+    char* line = NULL;
+    size_t len = 0, strip;
+    uint_fast8_t match = 0;
+
+    while (getline(&line, &len, file) != EOF)
+    {
+        if((strip = strspn(line, "\t ") > 0)) {
+            if(strip != len)
+                memmove(line, line + strip, len + 1 - strip);
+        }
+        if((match = sscanf(line, "Branch = %s", line)) > 0)
+            break;
```

Comment:
There is a ffGetPropValue for exactly this. It ignores tabs and spaces at the beginn of the line so it should work flawless.

Reply:
Ah. That's a recent addition I think. I forgot about that.